signer.go

```diff
@@ -3,7 +3,7 @@ package oauth
 import (
 	"crypto/rsa"
 	"errors"
-	"io/ioutil"
+	"io"
```

Comment:
Although changes looks good, the library build fails to import the io module. Can you please fix that?

Reply:
we will merge those changes directly

Reply:
We will incorporate these changes as part of this PR: https://github.com/Mastercard/oauth1-signer-go/pull/8